packages/card/event-card/style.module.css

```diff
@@ -0,0 +1,48 @@
+.pseudoCta {
+  --transition-duration: 250ms;
+  --transition-timing-function: ease-in-out;
+
+  margin-top: auto;
+  display: inline-flex;
+  align-items: center;
+  gap: 8px;
+  color: var(--wpl-neutral-500);
+}
+
+.pseudoCtaLabel {
+  font-weight: var(--font-weight-bold);
+  font-size: 16px;
+  line-height: 1.25;
+}
+
+.pseudoCtaIcon {
+  flex-shrink: 0;
+}
+
+.pseudoCtaIconArrow {
+  --translate-x: -4px;
+
+  transform: translateX(var(--translate-x));
+  transition-duration: var(--transition-duration);
+  transition-timing-function: var(--transition-timing-function);
+  transition-property: transform;
+  color: inherit;
+
+  @nest [data-testid='wpl-card']:hover & {
```

Comment:
Agreed, and I _would_ target the root class, but the root class in this case is in `<Card />` so I can't target it directly from `<EventCard />`'s CSS (right?). I think I could perhaps wrap `<EventCard />` in a root div, then target that?

Reply:
Makes me think we need to add className prop to base card.

Reply:
Definitely agree there, otherwise imagine the other workarounds we'd need to deal with for similar behavior in the future 😱

Reply:
or maybe we add a global className to the card that we can reference?

```tsx
<div className={classNames('wpl-card', s.card, s[appearance])} data-testid="wpl-card">
```

```css
@nest :global(.wpl-card):hover & {...}
```

Not validated that this works but just another idea.

Reply:
Hmm...I do like the global class option vs adding support for `className`. Using a global class will discourage making one-off modifications to cards' root CSS, which we probably want to avoid doing if we can help it, for styling/CSS consistency.

Reply:
Implemented the global class option in the latest two commits, works well I think!
https://github.com/hashicorp/react-components/pull/780/commits/5a155509289be30cf887cf15f69d5ae5c335ac4e
https://github.com/hashicorp/react-components/pull/780/commits/f3c76913e8bf3be35e607a6dc43de63b24ce01bb